stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -126,8 +129,12 @@ public ServerCall.Listener<ReqT> startCall(ServerCall<ReqT, RespT> call, Metadat
       Preconditions.checkArgument(
           call.getMethodDescriptor().getType().clientSendsOneMessage(),
           "asyncUnaryRequestCall is only for clientSendsOneMessage methods");
+      String preRpcAttempts = null;
+      if (headers.containsKey(GRPC_PREVIOUS_RPC_ATTEMPTS)) {
```

Comment:
Why do we need this check? `get` returns null if not present, so why can't we do :

```
String preRpcAttempts = headers.get(GRPC_PREVIOUS_RPC_ATTEMPTS);
```